src/runtime/mini_d3d12.h

```diff
@@ -489,15 +489,15 @@ typedef LONG RPC_STATUS;
 #endif
```

Comment:
This file appears to be just changed whitespace -- are there other changes?

Reply:
Tabs fixes are typically very small and they only creep in occasionally. Our PR process is high enough overhead in both human and buildbot time that I figured piggy backing a tiny white space change onto a smallish PR is fine.